index.ts

```diff
@@ -16,6 +16,14 @@
 
 'use strict';
 
+declare global {
+  namespace NodeJS {
```

Comment:
(disclosure: I may not understand namespaces). Is this necessary? The style guide forbids namespaces.

Reply:
AFAIK Yes - this block augments the `NodeJS` namespace defined in type definitions for Node. The alternative would to put this (and other declarations) in a `d.ts` file -- which I'm planning to do in a future PR.